### PR TITLE
added section for explaining stack choice of talawa

### LINKED ISSUE #450: Feature Request: Add to "Applications" section
**Is your feature request related to a problem? Please describe.**
Our applications page is missing detail on why we chose typescript, flutter, and monogodb. 

**Describe the solution you'd like**
Add to the "Applications" page information on why we used typescript, flutter, and monogodb. Our youtube page has videos that could be a helpful resource: https://www.youtube.com/@PalisadoesOrganization/videos. 

**Describe alternatives you've considered**
N/A

**Approach to be followed (optional)**
N/A

**Additional context**
N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by SalilHarit**
**What kind of change does this PR introduce?**

Added to the "Applications" page information on why talawa uses typescript, flutter, and monogodb in their projects

**Issue Number:**

Fixes  #450  

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

application section
Add snapshots or videos wherever possible.

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

<No 

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**

### CODE REVIEW

File: docs/applications/architecture.md [hunk 1]
```diff
@@ -30,4 +30,8 @@ This diagram outlines how the three Talawa components interact
 
 ![img](/img/Talawa.jpg)
 
+### Choosing the Right Stack
```
Comment (salman-farooqi) [inline docs/applications/architecture.md:33]: (It's just my opinion, I maybe am wrong)
From users' perspective;
The description for choosing the stack should be provided before defining each one of the following. You should tell why is it important to choose the right stack.
And secondly, separate points should be made for each stack before difining and describing it.
For example,
1. Typescript
2. Flutter
3. MongoDB
Thank You!

File: docs/applications/architecture.md [hunk 2]
```diff
@@ -30,4 +30,8 @@ This diagram outlines how the three Talawa components interact
...
+
+TypeScript's statically-typed nature and enhanced tooling capabilities provide increased reliability and maintainability to the codebase. Flutter's hot-reload feature allows for faster iteration and development of high-performance mobile applications with a native-like experience. Additionally, its support for both iOS and Android platforms makes it a versatile choice for cross-platform development. MongoDB's flexible document data model and scalability features make it a suitable choice for applications that require efficient data handling and storage. Its support for sharding and replication ensures high availability and data redundancy. Overall, the combination of TypeScript, Flutter, and MongoDB provides a robust and efficient stack for building modern and scalable applications.
+
```
Comment (salman-farooqi) [inline docs/applications/architecture.md:61]: `Its support for sharding and replication ensures high availability and data redundancy.`
This word is "sharding" or "sharing"?

Comment (salman-farooqi) [reply docs/applications/architecture.md:61]: I think your PR includes 'yarn.lock'. I, in my previous PR included this file mistakenly and was told by @palisadoes to remove it before making a PR.

**Comment by Cioppolo14** [comment]
@lonereaper Can you remove yarn.lock?  I believe if you update the version of docusaurus you are using to Talawa-Docs, there will be no differences and no then no yarn.lock needed.  Can you fix that? Thank you!

**Comment by SalilHarit** [comment]
> @lonereaper Can you remove yarn.lock?  I believe if you update the version of docusaurus you are using to Talawa-Docs, there will be no differences and no then no yarn.lock needed.  Can you fix that? Thank you!

Yes I will update it thanks for the review ! :)

File: docs/applications/architecture.md (hunk 1 above)
Comment (SalilHarit) [reply docs/applications/architecture.md:33]: Thanks for the review and for your suggestions I will incorporate these changes into my request

**Comment by Cioppolo14** [comment]
@lonereaper I'm going to close this PR. Please follow process and utilize our approving reviewers.  Thank you.

**Comment by SalilHarit** [comment]
I am so sorry I had no idea this would happen I new on GitHub

**Pull request closed without merging**